src/hydra.rs

```diff
@@ -59,6 +59,9 @@ pub struct JobsetOverview {
     pub project: String,
     pub name: String,
     pub nrfailed: i64,
+    pub starttime: Option<u64>,
+    pub lastcheckedtime: u64,
```

Comment:
I am _guessing_ that both `starttime` and `lastcheckedtime` are POSIX timestamps? It would be nice if this was explicit somehow. 

Not sure if this is idiomatic Rust code but..

```Rust
pub type posix_timestamp = u64
```
I wonder if we could use something like this?

Reply:
I introduced this typed. Ideally, we should use `SystemTime`. I tried but it seems we'd need to write a deserializer.